index.bs

```diff
@@ -7888,9 +7878,9 @@ macros:
 	tail-time: No
 </pre>
 
-The number of channels of the output corresponds to the number of
-channels of the [[mediacapture-streams#mediastreamtrack|MediaStreamTrack]]. If there is no valid
-audio track, then the number of channels output will be one silent
+The number of channels of the output corresponds to the number of channels of
```

Comment:
As best as I can tell, nothing really changed here except for line wrapping.  Please don't do that.  It makes review harder.

You can leave this, but please verify there were no changes.